### PR TITLE
Add a section about Slack to the participation guide

### PR DESCRIPTION
**Pull request by oleg-nenashev**

### CODE REVIEW

File: content/home/participate.md [hunk 1]
```diff
@@ -18,11 +18,19 @@ Also feel free to add yourself and/or your organization to
 This is a public listing for those individual and company contributors
 who declared their interest in the project.
 
+## Slack
+
+We plan to create the official Slack workspace later so that multiple workstreams and channels could be established there.
+For now, we have an `$openfeature` Slack channel on the CNCF Slack Workspace.
```
Comment (beeme1mr) [inline content/home/participate.md]:
```suggestion
For now, we have an `openfeature` Slack channel on the CNCF Slack Workspace.
```

File: content/home/participate.md [hunk 2]
```diff
@@ -18,11 +18,19 @@ Also feel free to add yourself and/or your organization to
...
+You can join this Slack using [this link](https://slack.cncf.io/).
+
 ## Project meetings
 
 Stay tuned for the announcements!
+**[Sign-up for news](https://bit.ly/openfeature-signup)** to get invite to the time poll and the meetings
```
Comment (beeme1mr) [inline content/home/participate.md]: This sentence is a bit confusing. Could you please rephrase it?

**Review by beeme1mr — CHANGES_REQUESTED** [review]

File: content/home/participate.md (hunk 2 above)
Comment (oleg-nenashev) [reply content/home/participate.md]:
```suggestion
**[Sign-up for news](https://bit.ly/openfeature-signup)** to get invited the meetings.
```

File: content/home/participate.md [hunk 3]
```diff
@@ -18,11 +18,19 @@ Also feel free to add yourself and/or your organization to
...
+
+We plan to create the official Slack workspace later so that multiple workstreams and channels could be established there.
+For now, we have an `openfeature` Slack channel on the CNCF Slack Workspace.
+You can join this Slack using [this link](https://slack.cncf.io/).
+
 ## Project meetings
 
 Stay tuned for the announcements!
+**[Sign-up for news](https://bit.ly/openfeature-signup)** to get invited the meetings.
+until the official channel is set up.
```
Comment (beeme1mr) [inline content/home/participate.md]: This section can be removed.

File: content/home/participate.md [hunk 4]
```diff
@@ -18,11 +18,19 @@ Also feel free to add yourself and/or your organization to
...
+## Slack
+
+We plan to create the official Slack workspace later so that multiple workstreams and channels could be established there.
+For now, we have an `openfeature` Slack channel on the CNCF Slack Workspace.
+You can join this Slack using [this link](https://slack.cncf.io/).
+
 ## Project meetings
 
 Stay tuned for the announcements!
+**[Sign-up for news](https://bit.ly/openfeature-signup)** to get invited the meetings.
```
Comment (beeme1mr) [inline content/home/participate.md]:
```suggestion
**[Sign-up](https://bit.ly/openfeature-signup)** to get invited the meetings.
```

**Pull request merged**